pkg/bundle/manager_test.go

```diff
@@ -320,7 +329,7 @@ func TestSortBundleNewestFirst(t *testing.T) {
 		}
 
 		bm := NewBundleManager(logr.Discard(), discovery, puller)
-		_ = bm.Update(bundle, true, allBundles)
+		bm.SortBundlesNewestFirst(allBundles)
```

Comment:
Not for this PR, but this is probably another place we can switch from time-based sorting to numeric, at least in the name. That is rename to something like `SortBundlesDescending` or something more like that.

Reply:
yeh, good idea